Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -1750,11 +1752,20 @@ static bool isCopyDisplayedPixelBufferAvailable()
     if (!updateLastPixelBuffer())
```

Comment:
`updateLastPixelBuffer()` will continue to return nil if the window isn't composited as it calls directly `[AVSampleBufferDisplayLayer copyDisplayedPixelBuffer]` ; shouldn't this be changed to call the VideoMediaSampleRenderer.mm version instead?